package.json

```diff
@@ -61,6 +61,7 @@
     "@react-navigation/drawer": "6.5.0",
     "@react-navigation/native": "6.0.13",
     "@react-navigation/stack": "6.3.1",
+    "@testing-library/react": "^13.4.0",
```

Comment:
No need for this package, please instead:

- Remove it
- Upgrade `@testing-library/react-native` to the latest version (so you can use the `screen` API in your test)

And use `@testing-library/react-native` instead.

Reply:
This change still needs to be made – use `@testing-library/react-native` instead of `@testing-library/react`

Reply:
@roryabraham actually `fireEvent.mouseOver` is not available in  `@testing-library/react-native` but is available in `@testing-library/react` that's why i didn't change it. 

Commented this yesterday maybe comment not posted as we have internet issue these days in Pakistan.